tests/serializers/tests.py

```diff
@@ -278,6 +278,12 @@ def test_deterministic_mapping_ordering(self):
             'second_category_pk': categories[1],
         })
 
+    def test_deserialize_force_insert(self):
+        """Tests that deserialized content can be saved with force_insert as a parameter."""
+        serial_str = serializers.serialize(self.serializer_name, Article.objects.all())
+        models = serializers.deserialize(self.serializer_name, serial_str)
+        [model.save(force_insert=False) for model in models]
```

Comment:
Hi Simon,
I've added an explicit for loop on branch
https://github.com/django/django/pull/4811/files and it does fail if
force_insert=True
I'm testing that we can pass the parameter more than the functionality of
force_insert.
Thanks
Michael

On Fri, Jun 5, 2015 at 4:11 PM, Simon Charette <notifications@github.com>
wrote:

> In tests/serializers/tests.py
> <https://github.com/django/django/pull/4811#discussion_r31820827>:
>
> > @@ -278,6 +278,12 @@ def test_deterministic_mapping_ordering(self):
> >              'second_category_pk': categories[1],
> >          })
> >
> > +    def test_deserialize_force_insert(self):
> > +        """Tests that deserialized content can be saved with force_insert as a parameter."""
> > +        serial_str = serializers.serialize(self.serializer_name, Article.objects.all())
> > +        models = serializers.deserialize(self.serializer_name, serial_str)
> > +        [model.save(force_insert=False) for model in models]
>
> I think an explicit for loop wouldn't hurt here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/django/django/pull/4811/files#r31820827>.
>



-- 
p: +353 87 6124111
e: michael@maithu.com